adal/src/androidTest/java/com/microsoft/aad/adal/AggregatedDispatcherTest.java

```diff
@@ -23,21 +23,27 @@
 
 package com.microsoft.aad.adal;
 
-import android.test.AndroidTestCase;
-import android.test.suitebuilder.annotation.SmallTest;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
 
 import java.net.MalformedURLException;
 import java.net.URL;
 
-public final class AggregatedDispatcherTest extends AndroidTestCase {
+import static org.junit.Assert.assertFalse;
```

Comment:
used on line 106. Since AndroidTestCase is a subclass of 'junit.framework.Assert', so it doesn't need to import these assert methods, but since we're deprecating the AndroidTestCase, so we need to add a lot of imports here. I will walk thru all these touched files to ensure no unnecessary imports. thanks